style.css

```diff
@@ -16267,3 +16267,44 @@ main.main-sidebar {
   opacity: 0.7;
   border-radius: 50%;
 }
+
+/* Header Accordian / Collapsible Headers */
+.accordion {
+  background-color: #a5aeb5;
+  cursor: pointer;
+  padding: 5px 0px 5px 15px;
+  width: 100%;
+  border: none;
+  text-align: left;
+  outline: none;
+  font-size: 22px;
+  color: white;
+  transition: 0.4s;
+  margin-bottom: 10px;
+}
+
+.active, .accordion:hover {
+  background-color: #455055;
+}
+
+.accordion:after {
+  content: "\25BC";
+  color: #EEE;
+  font-weight: bold;
+  font-size: 20px;
+  float: right;
+  margin-top: 3px;
+  margin-right: 15px;
+}
+
+.active:after {
+  content: "\25B2";
+}
+
+.panel {
+  padding: 0 18px;
+  background-color: white;
+  max-height: 0;
+  overflow: hidden;
+  transition: max-height 0.2s ease-out;
+}
```

Comment:
You shouldn't make any css changes on the `style.css` page directly. These changes should be done on `_custom.scss` and then you must go into the `bin` directory and run `compile.rb`. It will then update the `style.css` file.

Changing the styles on `.scss` files allows for more customization and extra features. You then have to run the `compile.rb` script to translate those changes into `css`.